paddle/fluid/framework/ir/graph_pattern_detector.cc

```diff
@@ -1658,6 +1658,24 @@ PDNode *patterns::DequantAny::operator()() {
   return dequant_out;
 }
 
+PDNode *patterns::MultipleQuantize::operator()() {
+  auto *prev_op = pattern->NewNode(prev_op_repr())->assert_is_op();
+
+  auto *prev_out = pattern->NewNode(prev_out_repr())->AsOutput();
+
+  // find operators with more than one quantization operator as output
+  prev_out->assert_more([&](Node *node) {
+    int counter = std::count_if(
+        node->outputs.begin(), node->outputs.end(),
+        [&](Node const *iter) { return iter->Name() == "quantize"; });
```

Comment:
Please add check if `iter` is an op node. I think it can be done like this:
```
return iter && iter->IsOp() && iter->Op()->Type() == "quantize";
```